scripts/pipeline-PREPARE.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+default_scripts="pipeline-BLOCKCHAIN.sh
```

Comment:
would it be better to add `set -ex` here so that you can see the commands that are being run?

Reply:
Eventually it would be nice not to run with that on by default (the open toolchain scripts don't as far as I recall) but I'll stick it in for now until getting better error handling in there